### PR TITLE
Clarifications to /sync description

### PR DESCRIPTION
**Pull request by DMRobertson**
### Pull Request Checklist

* [x] Pull request includes a [changelog file](https://github.com/matrix-org/matrix-doc/blob/master/CONTRIBUTING.rst#adding-to-the-changelog)
* [ ] Pull request includes a [sign off](https://github.com/matrix-org/matrix-doc/blob/master/CONTRIBUTING.rst#sign-off)
    * N/A element.io committer address
* [x] Pull request is classified as ['other changes'](https://github.com/matrix-org/matrix-doc/blob/master/CONTRIBUTING.rst#other-changes)

This
- helps to explain what is actually going on in these diagrams
- fixes an incorrect diagram which was nobbled in the move to markdown (#2980)

Preview: https://pr3534--matrix-org-previews.netlify.app

### CODE REVIEW

**Comment by turt2live** [comment]
Changelog please :)

**Comment by turt2live** [comment]
(we don't label PRs because it gets extremely confusing)

File: content/client-server-api/_index.md [hunk 1]
```diff
@@ -1424,7 +1424,9 @@ for each room, a `prev_batch` field, which can be passed as a `start`
 parameter to the [`/rooms/<room_id>/messages`](/client-server-api/#get_matrixclientv3roomsroomidmessages) API to retrieve earlier
 messages.
 
-You can visualise the range of events being returned as:
+For example, a `/sync` request might return a range of four events
+`E2`, `E3`,`E4` and `E5` within a given room, omitting two prior events
```
Comment (turt2live) [inline content/client-server-api/_index.md:1428]:
```suggestion
`E2`, `E3`, `E4` and `E5` within a given room, omitting two prior events
```

**Review by turt2live — APPROVED** [review]
Thanks! This does indeed feel a lot better to read.

File: content/client-server-api/_index.md [hunk 2]
```diff
@@ -1458,17 +1461,32 @@ containing only the most recent message events. A state "delta" is also
...
+                                 since: 'x-y-z'   |                  |                                                
+                                       prev_batch: 'd-e-f'       next_batch: 'u-v-w'
 ```
 
+The limited response includes a state delta which describes how the state
+of the room changes over the gap. This delta explains how to build the state
+prior to returned timeline (i.e. at `E7`) from the state the client knows 
+(i.e. at `E6`). To close the gap, the client should make a request to 
+[`/rooms/<room_id>/messages`](/client-server-api/#get_matrixclientv3roomsroomidmessages)
+with the query parameters `from=x-y-z` and `to=d-e-f`.
```
Comment (KitsuneRal) [inline content/client-server-api/_index.md:1488]: I think it should be the other way around, `from=d-e-f` and `to=x-y-z`? `/messages` returns events in the reverse order.

Comment (DMRobertson) [reply content/client-server-api/_index.md:1488]: Not sure, but FWIW [`/messages`](https://spec.matrix.org/v1.1/client-server-api/#get_matrixclientv3roomsroomidmessages) allows you to specify the direction as `f` or `b`.

Comment (KitsuneRal) [reply content/client-server-api/_index.md:1488]: Oh, right, I just routinely use it with `dir=b`. Maybe it's worth adding `dir=f` to avoid confusion.

**Pull request merged**